### ISSUE TITLE
How to get max size when I deliver `--capacityMB`?

### ISSUE THREAD

**Issue by no-reply-xx** [lenovo/python-redfish-lenovo#97]
Hi,
When I used `lenovo_create_raid_volume.py`, I wanted to use all space for the RAID volume, but I couldn't give a correct number of the max size, I tried `-1` and `10000000000`, they didn't work.
https://github.com/lenovo/python-redfish-lenovo/blob/master/examples/lenovo_create_raid_volume.py#L222

**Comment by no-reply-xx** [comment]
One more question: there is no `StripSize` option, how can I configure it in `lenovo_create_raid_volume.py` or `lenovo_update_raid_volume.py`?

**Comment by panyy3** [comment]
Thanks for your report. We have fixed the issue to allow using all space by pull https://github.com/lenovo/python-redfish-lenovo/pull/98.
For StripSize option, currently it can't configured yet.

**Comment by no-reply-xx** [comment]
@panyy3 thanks!
Do you have plan to support this? As max volume is fixed, I updated the issue title.

**Comment by panyy3** [comment]
Yes, StripSize option of Redfish API is planned to add the first half of next year.
